seshdash/views.py

```diff
@@ -996,6 +996,15 @@ def site_add_edit(request):
     return render(request, 'seshdash/settings/site_settings.html', context_dict)
 
 
+@login_required
+def delete_site(request, site_id):
+    """
+    Deletes a sesh site
+    """
+    site = Sesh_Site.objects.filter(id=site_id).first()
```

Comment:
How do we know the user has permission to delete the site? There needs to be a sort of permission check here.